ansto_simplon_api/config.py

```diff
@@ -0,0 +1,112 @@
+from functools import lru_cache
+from os.path import dirname as os_dirname, join as os_joinpath, realpath as os_realpath
+from pathlib import Path
+from typing import Annotated, Self
+
+from pydantic import Field, FilePath, GetPydanticSchema, SecretStr
+from pydantic_settings import (
+    BaseSettings,
+    PydanticBaseSettingsSource,
+    SettingsConfigDict,
+)
+
+__all__ = ("Settings", "get_settings")
+
+CUR_DIR = os_dirname(os_realpath(__file__))
+
+
+class APISettings(BaseSettings):
+    """API Settings"""
+
+    API_APP_NAME: str = Field(
+        title="App Name",
+        default="Simulated Simplon API",
+    )
+    API_DOCS_URL: str | None = Field(
+        title="OpenAPI Docs Path",
+        default=None,
```

Comment:
Why are we not adding default values for  `AS_API_DOCS_URL`, `API_REDOC_URL`, `AS_API_OPENAPI_URL`? If someone clones the repo and run the API, they wouldn't be able to access swagger unless they set these these values manually

Reply:
I've set the values in the example `docker-compose.yml` config.
These should be off by default, we want them disabled when we deploy the service, no easy way to do that with environment variables other than omitting them.

Reply:
can we have default values in this repo, and override them in the mx-simplon-api repo?